packages/mgt-components/src/components/BasePersonCardSection.ts

```diff
@@ -114,7 +114,7 @@ export abstract class BasePersonCardSection extends MgtTemplatedComponent {
    * @abstract
    * @memberof BasePersonCardSection
    */
-  public clearState(): void {
+  protected clearState(): void {
```

Comment:
@Mnickii this change is to support making the mgt-file-list use this as it's superclass